slack_cleaner/cli.py

```diff
@@ -104,11 +104,8 @@ def clean_channel(channel_id, time_range, user_id=None, bot=False):
             if m['type'] == 'message':
                 # Delete user messages
                 if m.get('user'):
-                    # No matter whos message, just delete
-                    if user_id is None:
```

Comment:
The problem is that when I run `slack-cleaner --token HIDDEN  --message --channel developer --bot` it also tries to delete everything from that channel lust because of I did't apply an optional argument. That's a dangerous thing and not what I am expecting. :scream: 

```
Will delete message -> marvinchiu : Welcome, let's make big thing
Will delete message -> james : Our API.
Will delete message -> james : <https://github.com/spe3d/>
396 message(s) will be cleaned.
```

Things like remove all can be done by Slack's `message retention` very easily. *slack-cleaner*. If you really want to keep the `user` argument, I'd suggest `--user=ALL` or `--user="*". Making things more explicit.

Really handy tool! 加油!